worlds/messenger/client_setup.py

```diff
@@ -0,0 +1,168 @@
+import io
+import logging
+import os.path
+import subprocess
+import urllib.request
+from pathlib import Path
+from shutil import which
+from tkinter.messagebox import askyesnocancel
+from typing import Any, Optional
+from zipfile import ZipFile
+from Utils import open_file
+
+import requests
+
+from Utils import is_linux, is_windows, messagebox, tuplize_version
+
+
+MOD_URL = "https://api.github.com/repos/alwaysintreble/TheMessengerRandomizerModAP/releases/latest"
+
+
+def launch_game(url: Optional[str] = None) -> None:
+    """Check the game installation, then launch it"""
+    if not (is_linux or is_windows):
```

Comment:
As below, this should also work on mac. There are a few more instance of that here.